.appveyor.yml

```diff
@@ -0,0 +1,108 @@
+  - SET ANSICON=121x90 (121x90)
+
+## Install PHP and composer, and run the appropriate composer command
+install:
+    - ps: |
+        # Check if installation is cached
+        if (!(Test-Path c:\tools\php)) {
+          appveyor-retry cinst --params '""/InstallDir:C:\tools\php""' --ignore-checksums -y php --version ((choco search php --exact --all-versions -r | select-string -pattern $env:php | sort { [version]($_ -split '\|' | select -last 1) } -Descending | Select-Object -first 1) -replace '[php|]','')
+          # install sqlite
+          appveyor-retry cinst -y sqlite
+          Get-ChildItem -Path c:\tools\php
+          cd c:\tools\php
+
+          # Set PHP environment items that are always needed
+          copy php.ini-production php.ini
+          Add-Content php.ini "`n date.timezone=UTC"
+          Add-Content php.ini "`n extension_dir=ext"
+          Add-Content php.ini "`n extension=php_openssl.dll"
+          Add-Content php.ini "`n extension=php_mbstring.dll"
+          Add-Content php.ini "`n extension=php_fileinfo.dll"
+          Add-Content php.ini "`n extension=php_pdo_sqlite.dll"
+          Add-Content php.ini "`n extension=php_sqlite3.dll"
+
+          # If needed get the MSSQL DLL's
+          if ($env:db -eq "mssql") {
+            $DLLVersion = "5.2.0rc1"
+            cd c:\tools\php\ext
+            $source = "https://windows.php.net/downloads/pecl/releases/sqlsrv/$($DLLVersion)/php_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip"
+            $destination = "c:\tools\php\ext\php_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip"
+            Invoke-WebRequest $source -OutFile $destination
+            7z x -y php_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip > $null
+            $source = "https://windows.php.net/downloads/pecl/releases/pdo_sqlsrv/$($DLLVersion)/php_pdo_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip"
+            $destination = "c:\tools\php\ext\php_pdo_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip"
+            Invoke-WebRequest $source -OutFile $destination
+            7z x -y php_pdo_sqlsrv-$($DLLVersion)-$($env:php)-nts-vc14-x64.zip > $null
+            Remove-Item c:\tools\php\* -include .zip
+            cd c:\tools\php
+            Add-Content php.ini "`nextension=php_sqlsrv.dll"
+            Add-Content php.ini "`nextension=php_pdo_sqlsrv.dll"
+            Add-Content php.ini "`n"
+          }
+
+          cd c:\projects\dbal
+
+          if (!(Test-Path c:\projects\dbal\composer.phar)) {
+            appveyor-retry appveyor DownloadFile https://getcomposer.org/composer.phar
+          }
+        }
+    # install composer dependencies
+    - appveyor-retry php composer.phar self-update
+    - appveyor-retry php composer.phar install --no-progress --profile
+
+before_test:
+# Selectively start the services
+  - ps: >-
+      if ($env:db -eq "mssql") {
+        $instanceName = $env:db_version.ToUpper()
+        Start-Service "MSSQL`$$instanceName"
```

Comment:
@photodude is there a reason we use `Start-Service` and not `net start`? As per the Appveyor's suggestion in https://github.com/appveyor/ci/issues/2204#issuecomment-376326756, it could be the reason for intermittent login failures.

Reply:
It's based on appveyor documentation https://www.appveyor.com/docs/services-databases/

If they have some new thing they suggest to do, their documentation does not reflect that.

Reply:
@photodude thanks. Trying to change this in #3071.